src/desktop/apps/collect2/server.tsx

```diff
@@ -6,14 +6,13 @@ import React from "react"
 import { Meta } from "./meta"
 import { buildServerAppContext } from "desktop/lib/buildServerAppContext"
 
-const app = (module.exports = express())
+export const app = (module.exports = express())
```

Comment:
I think `module.exports` needs to be removed here

Reply:
Ugh, yeah, good catch